packages/http-signature-utils/src/app.ts

```diff
@@ -0,0 +1,42 @@
+import Koa from 'koa'
+import Router from 'koa-router'
+
+import logger from 'koa-logger'
+import json from 'koa-json'
+import bodyParser from 'koa-bodyparser'
+
+import { createSignatureHeaders } from './utils/signatures'
+import { parseOrProvisionKey } from './utils/key'
+
+const app = new Koa()
+const router = new Router()
+
+// Load key
+const privateKey = parseOrProvisionKey(process.env.KEY_FILE)
+
+// Router
+router.get('/', async (ctx): Promise<void> => {
+  ctx.body = { msg: "I don't exist." }
+})
+
+router.post('/', async (ctx): Promise<void> => {
+  const { request, keyId } = ctx.request.body
+  if (!keyId || !request.headers || !request.method || !request.url) {
+    ctx.status = 400
+    return
+  }
+  const headers = await createSignatureHeaders({ request, privateKey, keyId })
+  ctx.body = headers
```

Comment:
does `status` get set automatically here?

Reply:
The status defaults to 200, see [here](https://koajs.com/#response).